src/codeu/chat/ServerMain.java

```diff
@@ -95,9 +95,9 @@ private static void runServer(Uuid id,
 
       try {
 
-        LOG.info("Established connection...");
+        //LOG.info("Established connection...");
```

Comment:
It is not recommended to submit commented out code. You have version control if you need to go back and look at your older versions.